pallets/capacity/src/lib.rs

```diff
@@ -284,47 +348,86 @@ pub mod pallet {
 			/// The amount of Capacity withdrawn from MSA.
 			amount: BalanceOf<T>,
 		},
+		/// The target of a staked amount was changed to a new MessageSourceId
+		StakingTargetChanged {
+			/// The account that retargeted the staking amount
+			account: T::AccountId,
+			/// The Provider MSA that the staking amount is taken from
+			from_msa: MessageSourceId,
+			/// The Provider MSA that the staking amount is retargeted to
+			to_msa: MessageSourceId,
+			/// The amount in token that was retargeted
+			amount: BalanceOf<T>,
+		},
+		/// Tokens have been staked on the network for Provider Boosting
+		ProviderBoosted {
+			/// The token account that staked tokens to the network.
+			account: T::AccountId,
+			/// The MSA that a token account targeted to receive Capacity based on this staking amount.
+			target: MessageSourceId,
+			/// An amount that was staked.
+			amount: BalanceOf<T>,
+			/// The Capacity amount issued to the target as a result of the stake.
+			capacity: BalanceOf<T>,
+		},
 	}
 
 	#[pallet::error]
 	pub enum Error<T> {
 		/// Staker attempted to stake to an invalid staking target.
 		InvalidTarget,
 		/// Capacity is not available for the given MSA.
-		InsufficientBalance,
+		InsufficientCapacityBalance,
 		/// Staker is attempting to stake an amount below the minimum amount.
-		InsufficientStakingAmount,
-		/// Staker is attempting to stake a zero amount.
+		StakingAmountBelowMinimum,
+		/// Staker is attempting to stake a zero amount.  DEPRECATED
```

Comment:
It is still used by `ensure_can_stake`, but you can tag it with a rust `deprecated` attribute as well

```suggestion
                /// Staker is attempting to stake a zero amount.
		/// #[deprecated(since = "1.13.0", note = "Use ?? instead")]

```